packages/launchpad/cypress/e2e/global-mode.cy.ts

```diff
@@ -144,6 +144,36 @@ describe('Launchpad: Global Mode', () => {
       cy.get('[data-cy="dropzone"]').should('not.exist')
     })
 
+    it('updates breadcrumb when selecting a project and navigating back', () => {
+      const getBreadcrumbLink = (name: string, options: { disabled: boolean } = { disabled: false }) => {
+        return cy.findByRole('link', { name }).should('have.attr', 'aria-disabled', options.disabled ? 'true' : 'false')
+      }
+
+      const projectList = ['todos']
+
+      setupAndValidateProjectsList(projectList, ['--e2e'])
```

Comment:
Do we need the `--e2e` flag since we are opening in global mode?

Reply:
Addressed in [bdb5a22](https://github.com/cypress-io/cypress/pull/21297/commits/bdb5a221dfd3cd6e700840bbe99acbe3ff28c0f0)